json2html/jsonconv.py

```diff
@@ -5,11 +7,13 @@
 (c) Varun Malhotra 2013
 Source Code: https://github.com/softvar/json2html
 ------------
+Michel Müller 2015-1-31: Added bootstrap option, converting table-like JSON
```

Comment:
I appreciate the effort and would always like the contribution/patch to be mentioned.
How about just writing
```
Contributors: Michel Müller, patch #2 - https://github.com/softvar/json2html/pull/2
```

You can commit your changes and push it, no need to open a new PR.
BDW, thanks a lot for the great work. :+1: